README.md

```diff
@@ -81,6 +81,10 @@ To run the project, execute the following command:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Deployment
+
+You can deploy this project using: https://sam2if.github.io/
+
```

Comment:
- [OPTIONAL] There is a 'Live Demo' section in your README file, it would be better to use only one section for the purpose of deployment so this is less confusing for the user.